Orange/widgets/data/owcorrelations.py

```diff
@@ -365,7 +365,11 @@ def set_data(self, data):
 
     def set_feature_model(self):
         self.feature_model.set_domain(self.cont_data and self.cont_data.domain)
-        self.feature = None
+        data = self.data
+        if data and data.domain.has_continuous_class and self.cont_data:
```

Comment:
Is it necessary to test for `data` or would `if self.cont_data and data.domain.has_continuous_class` suffice?

If you don't change it by the end of the week (or explain why the test is necessary), I will consider my comment nitpicking and merge the code as it is. :)

Reply:
I removed the unnecessary condition.

Reply:
I'd do it myself but wasn't sure about it. :)